packages/at_contact/lib/src/at_contacts_impl.dart

```diff
@@ -418,11 +416,11 @@ class AtContactsImpl implements AtContactsLibrary {
     }
     var atKey = _formKey(KeyType.group, key: atGroup.groupId!);
     // Add all contacts in atContacts from atGroup
-    atContacts.forEach((contact) {
+    for (var contact in atContacts) {
```

Comment:
could be `final`

Reply:
Changed. Changed a few other for loops also.